XenServerHealthCheck/ProjectInstaller.Designer.cs

```diff
@@ -33,9 +33,9 @@ private void InitializeComponent()
             // 
             // XenServerHealthCheckInstaller
             // 
-            this.XenServerHealthCheckInstaller.Description = Branding.COMPANY_NAME_SHORT + " " + Branding.PRODUCT_BRAND + " Health Check";
-            this.XenServerHealthCheckInstaller.DisplayName = Branding.COMPANY_NAME_SHORT + " " + Branding.PRODUCT_BRAND + " Health Check";
-            this.XenServerHealthCheckInstaller.ServiceName = Branding.PRODUCT_BRAND + "HealthCheck";
+            this.XenServerHealthCheckInstaller.Description = Branding.COMPANY_AND_PRODUCT_BRAND + " Health Check";
+            this.XenServerHealthCheckInstaller.DisplayName = Branding.COMPANY_AND_PRODUCT_BRAND + " Health Check";
+            this.XenServerHealthCheckInstaller.ServiceName = string.Join("", Branding.PRODUCT_BRAND.Split(' ')) + "HealthCheck";

```

Comment:
Please note that the HEALTHCHECKSERVICENAME in https://github.com/xenserver/xenadmin/blob/master/XenModel/Actions/HealthCheck/TransferHealthCheckSettingAction.cs#L47 needs to match this ServiceName, otherwise XenCenter won't find the service to send updates to. 
I don't think we even need to build the ServiceName here, we could just hardcode it in both places to the same value, or we could define a variable in branding and use it in these two places.

Reply:
You avoided a big bug here. Thank you. I would hardcode it.
BTW, in WixInstaller/XenCenter.wxs we also specify the service name, what's the relationship between them?